packages/gatsby/cache-dir/head/head-export-handler-for-browser.js

```diff
@@ -96,7 +113,11 @@ export function headHandlerForBrowser({
       const { render } = reactDOMUtils()
 
       const HeadElement = (
-        <pageComponent.Head {...filterHeadProps(pageComponentProps)} />
+        <IsHeadRenderContext.Provider value={true}>
+          <svg data-gatsby-head-react-19-workaround="true">
```

Comment:
`<svg>` is used as it allows to escape React 19 automatic hoisting of `<head>` elements to actual `<head>` in document and allow our handling.

If we are patching `React.createElemenent` for `html` and `body` attributes - might be good idea to use same setup for tags allowed in Head as well and we could avoid this pretty hacky `<svg>` escape hatch and limit number of hacks that are employed to get React 19 compatbility